### DISCUSSION TITLE
How to search  variable  in the variables window?

### DISCUSSION THREAD

**Discussion by wilikeit** [pyscripter/pyscripter#1327] [Q&A]
I click on the search icon, and there did not appear a box for me to type in a keyword
![image](https://github.com/user-attachments/assets/08a21caa-1455-42dd-bf3f-b2a510901e34)

**Comment by pyscripter** [comment]
This is not a search icon.   It relates to https://pyscripter.blogspot.com/2023/10/feature-preview-debug-inspectors.html.

**Reply by wilikeit** [reply]
The magnifying glass can examine variables in detail, but if there are too many variables, can it search, like the following, does not currently have this function?

![image](https://github.com/user-attachments/assets/7d18deb7-1135-4f3c-8f48-c0a84e285b71)

**Reply by pyscripter** [reply]
The Variables window supports incremental search.  Click on the Name column and start typing the name of what you are looking for.

This is also the case for all tree based controls (File explorer, Code Explorer etc.)

**Reply by wilikeit** [reply]
I see, this usage I often use in WINDOWS Explorer, entering the first letter will automatically locate the item at the beginning of the letter, to some extent, it is also a search function

**Reply by wilikeit** [reply]
But I think it would be better to have a dedicated search box for that
